File: apps/problems/max-element/boilerplate-full/function.cpp

#include <iostream>
#include <vector>
#include <string>
#include <fstream>
#include <climits>

##USER_CODE_HERE##

int main() {
  int size_arr;
  std::ifstream inputFile("/sandbox/input.txt");
  inputFile >> size_arr;
  std::vector<int> arr(size_arr);
  for(int i = 0; i < size_arr; ++i) inputFile >> arr[i];
  int result = maxElement(arr);
  std::cout << result << std::endl;
  return 0;
}
    

File: apps/problems/max-element/boilerplate-full/function.rs

use std::fs::File;
use std::io::{self, BufRead};
use std::path::Path;

##USER_CODE_HERE##

fn main() {
let path = Path::new("/sandbox/input.txt");
let file = File::open(path).unwrap();
let reader = io::BufReader::new(file);
let mut lines = reader.lines();
let size_arr: usize = input.next().unwrap().parse().unwrap();
let arr: Vec<i32> = input.take(size_arr).map(|s| s.parse().unwrap()).collect();
let result = maxElement(arr);
println!("{}", result);
}
  